### ISSUE TITLE
Add back the QR code download button 

### ISSUE THREAD

**Issue by lakchote** [Expensify/App#40110]
QR code download doesn't work anymore (as of today) since the new arch (Fabric [PR](https://github.com/Expensify/App/pull/13767)) got merged.

TL;DR: `react-native-view-shot` does not support new arch yet.

It was decided [here](https://expensify.slack.com/archives/C03TQ48KC/p1712840704611719?thread_ts=1712838112.030309&cid=C03TQ48KC) to hide the Download button for QR code in the meantime, to prevent users from encountering bugs and degrading user experience.

When the update 0.74 for `react-native` is available, we should: 
- use it so `react-native-view-shot` works again as intended
- Revert PR [here](https://github.com/Expensify/App/pull/40109) and show back the Download button.
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01d1641a383b71483e</li>
        <li>Upwork Job ID: 1778781286743756800</li>
        <li>Last Price Increase: 2024-04-12</li>
    </ul>
</details>

**Comment by lakchote** [comment]
Still waiting for RN 0.74 to be effective. Tracking PR [here](https://github.com/Expensify/App/issues/37374).

**Comment by lakchote** [comment]
> Still waiting for RN 0.74 to be effective. Tracking PR [here](https://github.com/Expensify/App/issues/37374).

Still waiting for RN 0.74 PR to be available. Putting this as a monthly in the meantime.

**Comment by ahmedGaber93** [comment]
Not overdue, Still hold on #37374

**Comment by lakchote** [comment]
> Not overdue, Still hold on #37374

Same.

**Comment by ahmedGaber93** [comment]
Not overdue, Still hold on https://github.com/Expensify/App/issues/37374

**Comment by ahmedGaber93** [comment]
I will unassign me to clean up my k2.

**Comment by s77rt** [comment]
Alternative solution: [`@s77rt/react-native-viewshot`](https://github.com/s77rt/react-native-viewshot)

**Comment by lakchote** [comment]
PR https://github.com/Expensify/App/issues/37374 was merged, will take a look to add it back as soon as I can.

**Comment by lakchote** [comment]
PR didn't solve the issue, see https://github.com/Expensify/App/issues/49868#issuecomment-2381918469. We're going to wait for a new stable version.

**Comment by brunovjk** [comment]
I don't have a physical iOS device, I only have Android. I'm available to test if necessary. Thanks.

**Comment by lakchote** [comment]
Still waiting for a stable version of `react-native-viewshot` `v4.0.0` (see https://github.com/Expensify/App/issues/49868#issuecomment-2381918469).

**Comment by lakchote** [comment]
> Still waiting for a stable version of `react-native-viewshot` `v4.0.0` (see [#49868 (comment)](https://github.com/Expensify/App/issues/49868#issuecomment-2381918469)).

Stable version is now available. As such, I've raised a new PR [here](https://github.com/Expensify/App/pull/53757).

**Comment by jliexpensify** [comment]
@lakchote @fedirjh - heads up that I am OOO from 19th - 29th Dec. I'm happy to still be assigned and can sort out any payments when I'm back, but feel free to unassign me and reassign someone else if any urgent action is needed from B0.

**Comment by lakchote** [comment]
@jliexpensify please process payment for @fedirjh for the C+ review.

If the timing is too short for you no worries, let me know and I'll reassign.

**Comment by jliexpensify** [comment]
Easy, here's a Payment Summary:

- C+: @fedirjh $250 (to be paid via New.Dot)

@lakchote @fedirjh if there's a checklist, feel free to add that and ic an get to it when I'm back.

Otherwise this can be closed?

**Comment by jliexpensify** [comment]
Cool, feel free to request on ND then - closing!
